contracts/Front/MultiSig.sol

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright: Ambrosus Inc.
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+pragma solidity ^0.4.23;
+
+import "multisig-wallet-gnosis/contracts/MultiSigWallet.sol";
+
+import "../Configuration/Config.sol";
+import "../Boilerplate/Head.sol";
+import "../Boilerplate/Multiplexer.sol";
+
+
+contract MultiSig is Base, MultiSigWallet {
```

Comment:
Ivan Rukhavets [4:55 PM]
1. The multiplexer should not be a part of context and not a Base class

Reply:
MultiSigWallet.sol added separatly out of context.